pkg/convert/json_ngxtranslate_test.go

```diff
@@ -0,0 +1,129 @@
+package convert
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+)
+
+func Test_fromNgxTranslate(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name       string
+		serialized []byte
+		expected   model.Messages
+		wantErr    bool
```

Comment:
Unused

Reply:
What do you mean by unused? We have more than one test so we have a struct array and we are using mentioned properties in for loop to run all tests

Reply:
`wantErr` is defined in struct, but unused in actual tests